package.json

```diff
@@ -102,24 +102,35 @@
     ],
     "rules": {
       "angular/angularelement": "error",
-      "angular/di": "error",
-      "angular/di-order": "error",
+      "angular/di": "warn",
+      "angular/di-order": "warn",
       "angular/di-unused": "error",
       "angular/document-service": "error",
       "angular/log": "error",
       "angular/module-dependency-order": "error",
       "angular/no-service-method": "error",
-      "angular/no-services": "error",
+      "angular/no-services": "warn",
       "angular/window-service": "error",
       "compat/compat": "error",
       "arrow-parens": "off",
       "constructor-super": "off",
       "generator-star-spacing": "off",
+      "max-len": [
+        "error",
+        {
+          "code": 120,
```

Comment:
I'm in favor of the expanded line width. Given the size of modern monitors, and the fact that we don't code on punch cards anymore - my .02 is that an 80 character limit feels archaic.